src/components/AuthForm.js

```diff
@@ -22,8 +22,8 @@ class AuthForm extends CommunisComponent {
     this.handleSubmit = this.handleSubmit.bind(this);
     this.state = {
       form: {
-        username: null,
-        password: null,
+        username: process.env.REACT_APP_DEV_USER,
+        password: process.env.REACT_APP_DEV_PASSWORD,
```

Comment:
Nice! These will be environment variables

Reply:
You know you can try and find a new username.https://help.github.com/en/enterprise/2.13/user/articles/changing-your-github-username